configs/original_dst_proxy.json

```diff
@@ -0,0 +1,44 @@
+{
+    "listeners": [{
+        "address": "tcp://0.0.0.0:10000",
+	"use_original_dst": true,
+        "filters": [{
+            "type": "read",
+            "name": "http_connection_manager",
+            "config": {
+                "codec_type": "auto",
+                "stat_prefix": "ingress_http",
+                "route_config": {
+                    "virtual_hosts": [{
+                        "name": "local_service",
+                        "domains": [
+                            "*"
+                        ],
+                        "routes": [{
+                            "timeout_ms": 0,
+                            "prefix": "/",
+                            "cluster": "original_dst_cluster"
```

Comment:
cluster name is not semantically important, I'll change the config example to reflect this.